### PR TITLE
Bugfix #83

### PR DESCRIPTION
**Pull request by gkumbhat**
Added the bugfix for the citation.xml read error.
Changed the dfr.py file.

### CODE REVIEW

File: tethne/readers/dfr.py [hunk 1]
```diff
@@ -214,7 +214,7 @@ def read(path, corpus=True, index_by='doi', **kwargs):
        >>> from tethne.readers import dfr
        >>> papers = dfr.read("/Path/to/DfR")
     """
-    parser = DfRParser(os.path.join(path, "citations.XML"))
+    parser = DfRParser(os.path.join(path, "citations.xml"))
```
Comment (erickpeirson) [inline tethne/readers/dfr.py]: What happens if `citations.xml` is missing, but `citations.XML` is present?

**Comment by erickpeirson** [comment]
@gkumbhat Oops, looks like your tests are failing! Did you test this before creating a pull request?

File: tethne/readers/dfr.py (hunk 1 above)
Comment (gkumbhat) [inline tethne/readers/dfr.py]: Actually by default "citations.xml" is generated when we export it.

**Comment by gkumbhat** [comment]
Yeah I did check it and it was running. Let me check it one more time.

File: tethne/readers/dfr.py (hunk 1 above)
Comment (erickpeirson) [inline tethne/readers/dfr.py]: Older versions of DfR had files called `citations.XML`. One of the test datasets has a file by that name...

**Comment by erickpeirson** [comment]
Travis never lies. :-) Click on "Details" next to `continuous-integration/travis-ci/push` in the CI block toward the bottom of this page.

**Comment by gkumbhat** [comment]
Got the reason why it failed. I actually changed my dataset and it was having file "citations.xml" and Travis is having dataset which has "citations.XML". I will change the code to accept both.

**Comment by erickpeirson** [comment]
Great work!

**Pull request merged**